src/Test/Plutip/Internal/LocalCluster.hs

```diff
@@ -72,14 +72,18 @@ startCluster ::
   IO (TVar (ClusterStatus a), a)
 startCluster conf onClusterStart = do
   status <- newTVarIO ClusterStarting
+  tid <- myThreadId
   void $
     forkFinally
       ( withPlutusInterface conf $ \clusterEnv -> do
           res <- runReaderT onClusterStart clusterEnv
           atomically $ writeTVar status (ClusterStarted res)
           atomically $ readTVar status >>= \case ClusterClosing -> pure (); _ -> retrySTM
       )
-      (either (error . show) (const (atomically (writeTVar status ClusterClosed))))
+      ( \e -> do
```

Comment:
I associate the letter `e` to error, but this is actually the result (Either error response). A better variable name could make that more obvious.

Reply:
completely agree, will change this once nix allows me to enter the project

Reply:
changed to `result`